src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2414,12 +2426,6 @@
           "type": "string"
         },
 
-        "markdownMessageMimeType": {
```

Comment:
>markdownMessageMimeType [](start = 9, length = 23)

@lgolding, can you recall for me why we deleted this? simply can't remember the rationale (i do recall the decision)

Reply:
Closing loop: Because we only support Markdown, we specify GFM, and CommonMark is a subset of GFM.

---
In reply to: [275446870](https://github.com/Microsoft/sarif-sdk/pull/1426#discussion_r275446870) [](ancestors = 275446870)